haskell/nixpkgs.bzl

```diff
@@ -159,7 +162,9 @@ _ghc_nixpkgs_toolchain = repository_rule(_ghc_nixpkgs_toolchain_impl)
 
 def haskell_register_ghc_nixpkgs(
         version,
-        is_static = False,
+        is_static = None, # DEPRECATED. See _check_static_attributes_compatibility.
+        static_runtime = None,
+        fully_static_link = None,
```

Comment:
I think it would be good to add descriptions for these attributes to the `Args` section of the docstring. In particular, that `is_static` is deprecated, that `static_runtime = True` means that GHC has to be built with a static RTS and that it's needed to use static Haskell libs for GHCi/TH, and that `fully_static_link` will produce fully statically linked binaries.